website/app/styles/pages/application/header.scss

```diff
@@ -139,75 +135,35 @@
   }
 }
 
-.doc-page-header__nav-item-generic {
```

Comment:
These styles were only affecting the support icon link and the github link and the only thing they did were add display block and make the focus ring look way thicker than the other nav items' focus states.

![Screenshot 2024-11-21 at 11 17 36 AM](https://github.com/user-attachments/assets/74de7064-b149-4538-88f5-b6b0364fbf71)